activemodel/lib/active_model/dirty.rb

```diff
@@ -180,7 +180,7 @@ def attribute_changed?(attr, options = nil) #:nodoc:
         result &&= options[:to] == __send__(attr) if options.key?(:to)
         result &&= options[:from] == changed_attributes[attr] if options.key?(:from)
       end
-      result
+      !!result
```

Comment:
Although I did just notice the commit message is pretty barren. @Sen-Zhang can you try to provide more context in your commit messages in the future?